frontend/language/src/nb.json

```
@@ -800,6 +800,7 @@
   "process_editor.configuration_panel_custom_receipt_textfield_label": "Navn på sidegruppe",
   "process_editor.configuration_panel_data_model_selection_description": "Velg en datamodell å knytte til prosessteget",
   "process_editor.configuration_panel_data_task": "Oppgave: Utfylling",
+  "process_editor.configuration_panel_data_types_to_sign_required": "Du må minst velge en datatype",
```

Hva med å endre til dette? 
```suggestion
  "process_editor.configuration_panel_data_types_to_sign_required": "Du må velge minst en datatype",
```

---

Done!